docs/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -765,28 +1115,39 @@ Model Configuration Parameters
+   Flag that determines whether to use the inline post option, which calls the Unified Post Processor (:term:`UPP`) from within the UFS Weather Model. The default ``WRITE_DOPOST: false`` does not use the inline post functionality, and the ``run_post`` tasks are called from outside of the UFS Weather Model. If ``WRITE_DOPOST: true``, the ``WRITE_DOPOST`` flag in the ``model_configure`` file will be set to true, and the post-processing (:term:`UPP`) tasks will be called from within the Weather Model. This means that the post-processed files (in :term:`grib2` format) are output by the Weather Model at the same time that it outputs the ``dynf###.nc`` and ``phyf###.nc`` files. Setting ``WRITE_DOPOST: true`` turns off the separate ``run_post`` task in ``setup.py`` to avoid unnecessary computations. Valid values: ``True`` | ``False``
 
 Computational Parameters
 ----------------------------
 
-``LAYOUT_X, LAYOUT_Y``: (Default: "")
-   The number of :term:`MPI` tasks (processes) to use in the two horizontal directions (x and y) of the regional grid when running the forecast model.
+``LAYOUT_X``: (Default: ``'{{ LAYOUT_X }}'``)
+   The number of :term:`MPI` tasks (processes) to use in the x direction of the regional grid when running the forecast model.
+
+``LAYOUT_Y``: (Default: ``'{{ LAYOUT_Y }}'``)
+   The number of :term:`MPI` tasks (processes) to use in the y direction of the regional grid when running the forecast model.
 
-``BLOCKSIZE``: (Default: "")
+``BLOCKSIZE``: (Default: ``'{{ BLOCKSIZE }}'``)
    The amount of data that is passed into the cache at a time.
 
 .. note::
 
    In ``config_defaults.yaml`` the computational parameters are set to null strings so that:
+.. COMMENT: Is this still true^ 
 
    #. If the experiment is using a predefined grid and the user sets the parameter in the user-specified experiment configuration file (i.e., ``config.yaml``), that value will be used in the forecast(s). Otherwise, the default value for that predefined grid will be used.
    #. If the experiment is *not* using a predefined grid (i.e., it is using a custom grid whose parameters are specified in the experiment configuration file), then the user must specify a value for the parameter in that configuration file. Otherwise, the parameter will remain set to a null string, and the experiment generation will fail because the generation scripts check to ensure that all the parameters defined in this section are set to non-empty strings before creating the experiment directory.
```

Comment:
It's probably fine to remove this section. As you point out it's not strictly true anymore, and it's just overkill information for the users guide

```suggestion
```